pkg/util/sloconfig/colocation_config.go

```diff
@@ -43,6 +43,7 @@ func DefaultColocationCfg() configuration.ColocationCfg {
 
 func DefaultColocationStrategy() configuration.ColocationStrategy {
 	calculatePolicy := configuration.CalculateByPodUsage
+	var defaultMemoryCollectPolicy slov1alpha1.MemoryCollectPolicy = "usageWithoutPageCache"
```

Comment:
As said above, please define a `const MemoryCollectPolicyWithoutPageCache = "usageWithoutPageCache"`, instead of hard-coding the policy string in each comparison.